e2e/manifests/devnet0.toml

```diff
@@ -2,6 +2,6 @@
 # This is aimed at local dev environments.
 
 network = "devnet"
-anvil_chains = ["mock_l1", "mock_l2"]
+anvil_chains = ["mock_l1", "mock_op", "mock_arb"]
```

Comment:
can remove `mock_l1` I think? Not sure devs need it, and it uses additional resources...

Reply:
agreed. have added a conditional to avs deployment step